source/manual/cdn.html.md

```diff
@@ -180,49 +143,36 @@ done
 [firewall rules]: https://github.com/alphagov/govuk-provisioning/blob/master/vcloud-edge_gateway/vars/production_carrenza_vars.yaml
 [vcl_config]: https://github.com/alphagov/govuk-cdn-config/
 [check-cdn-ip-ranges]: https://deploy.publishing.service.gov.uk/job/Check_CDN_IP_Ranges/
-[raise-with-re]: raising-issues-with-reliability-engineering.html
+[raise-with-re]: https://docs.publishing.service.gov.uk/manual/raising-issues-with-reliability-engineering.html
```

Comment:
```suggestion
[raise-with-re]: /manual/raising-issues-with-reliability-engineering.html
```

Links to dev docs tend to not have the domain in them. (We should do the same below).

Reply:
I've corrected for both links.